pkg/minibroker/rabbitmq.go

```diff
@@ -45,6 +45,19 @@ func (p RabbitmqProvider) Bind(services []corev1.Service, params map[string]inte
 		return nil, errors.Errorf("no amqp port found")
 	}
 
+	rabbitmqParams, ok := params["rabbitmq"].(map[string]interface{})
+	if !ok {
+		rabbitmqParams = make(map[string]interface{})
+	}
+
+	var username string
+	usernameVal, ok := rabbitmqParams["username"]
+	if ok {
+		username = usernameVal.(string)
```

Comment:
What happens if `username` is set to `3` (i.e. not a string)?

Reply:
![image](https://user-images.githubusercontent.com/3149049/89476160-96c36f00-d74f-11ea-9693-84f40022b1bb.png)